crt/debugmalloc.c

```diff
@@ -0,0 +1,538 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>
+
+#include <myst/spinlock.h>
```

Comment:
Are the myst includes intentional?

Reply:
Yes, but I removed ``debugmalloc.c`` from crt and now crt uses the one that is in the kernel.